src/utilities/hooks/ModalPaymentContext/constants.ts

```diff
@@ -6,19 +6,27 @@
  */
 const SET_MODAL_VIEW = 'SET_MODAL_VIEW';
 const SET_AMOUNT = 'SET_AMOUNT';
+const SET_CUSTOM_INPUT = 'SET_CUSTOM_INPUT';
+const SET_COVERED_BY_CUSTOMER = 'SET_COVERED_BY_CUSTOMER';
 const CLOSE_MODAL = 'CLOSE_MODAL';
 const CLEAR_FORMS = 'CLEAR_FORMS';
 
 const SET_SELLER_DATA = 'SET_SELLER_DATA';
 const UPDATE_SELLER_DATA = 'UPDATE_SELLER_DATA';
 
 const EMAIL_REGEX = /[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$/;
+
+const TRANSACTION_FEE_PERCENT = 0.03;
```

Comment:
The actual fee is 2.9% + $.30. We should account for the exact amount, rounded up to the nearest cent. cc @nanxiy would you prefer cent or dollar?

Reply:
btw in the future we'll add a Fees endpoint that will return the fees that are associated with the payment cc @Jeremy-Xue

Reply:
I think rounded is fine, 3%

Reply:
From Square: 
You can send an unlimited number of invoices for a fee of 2.9% + 30¢. Fees are applied at the time an invoice is paid by your customer with their credit or debit card, Apple Pay, or Google Pay.

We should get rid of credit card in the sentence since all transactions incur a fee of 2.9% + 30 cents.

Reply:
@nanxiy rounded to cent or dollar?

Reply:
round to cent - I like the transparency

Reply:
rounded up btw